lucid/modelzoo/vision_base.py

```diff
@@ -200,6 +201,14 @@ def import_graph(self, t_input=None, scope='import', forget_xy_shape=True, input
         self.graph_def, final_input_map, name=scope)
     self.post_import(scope)
 
+    def T(layer):
```

Comment:
The input case can't be fully supported here without some more changes and is kind of idiosyncratic. With that said, I did refactor `T` on the render side so that it just adds on functionality to the version from here.